Source/WarpX.H

```diff
@@ -379,11 +379,11 @@ public:
     amrex::Vector< std::unique_ptr<NCIGodfreyFilter> > nci_godfrey_filter_exeybz;
     amrex::Vector< std::unique_ptr<NCIGodfreyFilter> > nci_godfrey_filter_bxbyez;
 
-    amrex::Real time_of_last_gal_shift  = 0;
-    amrex::Vector<amrex::Real> m_v_galilean{amrex::Vector<amrex::Real>(3, amrex::Real(0.))};
+    amrex::Real time_of_last_gal_shift = 0;
+    amrex::Vector<amrex::Real> m_v_galilean = amrex::Vector<amrex::Real>(3, amrex::Real(0.));
     amrex::Array<amrex::Real,3> m_galilean_shift = {{0}};
 
-    amrex::Vector<amrex::Real> m_v_comoving{amrex::Vector<amrex::Real>(3, amrex::Real(0.))};
+    amrex::Vector<amrex::Real> m_v_comoving = amrex::Vector<amrex::Real>(3, amrex::Real(0.));
```

Comment:
I am confused why `amrex::Vector<amrex::Real> m_v_comoving{amrex::Vector<amrex::Real>(3, amrex::Real(0.))};` does not work.  The stackoverflow's example that works is
```
class Foo {
    vector<int> val{vector<int>(5,0)};
};
```
They look similar to me.

Reply:
You are right, only `()` would be wrong.
I guess that's another NVCC C++17 bug then, (besides #2596).